src/macros.jl

```diff
@@ -15,15 +15,7 @@ function sub_val(ex::Expr, m::Module)
     elseif ex.head == :kw
         ex.args[2] = sub_val(ex.args[2], m)
     end
-    if ex.head == :call
-        try
-            eval(m, ex)
-        catch MethodError e
-            return ex
-        end
-    else
-        return ex
-    end
+    return ex
```

Comment:
@tshort This was there because if I had e.g. `sampleon(x, y)` I would want the whole expression to be a single input, without this, this will call `sampleon` with values of signals `x` and `y` which does not mean anything.

Was there some other reason you removed these lines?